### DISCUSSION TITLE
Update from 9.2 to 11.0 RepeatUntil is missing

### DISCUSSION THREAD

**Discussion by bwienefoet** [Gremlinq/ExRam.Gremlinq#820] [Q&A] [answered]
Hi @all,

i'm trying an update from 9.2 to Version 11.0 of Gremlinq but after that all my querys with RepeatUntil and Emit wont work anymore.
RepeatUntil and Emit are missing.

Example Query: 
var query = G.V(vertexId)
            .RepeatUntil(__ =>
                    __.InE<Assigned>().OutV(),
                __ => __.InE<Assigned>().OutV<User>().Where(x => x.Id == userId))
            .Emit().Cast<Role>().Where(r => r.ValidFrom <= DateTime.UtcNow && r.ValidUntil >= DateTime.UtcNow);

Where is RepeatUntil in Version 11.0

**Comment by danielcweber** [comment, marked as answer]
It's now done through a builder like in [this example](https://github.com/ExRam/ExRam.Gremlinq/blob/1b062326cd538da3a4f7fd826f75c362f848f472/test/ExRam.Gremlinq.Core.Tests/QueryExecutionTest.cs#L3711).
